packages/postcss/src/esm.ts

```diff
@@ -135,14 +135,16 @@ export function createPlugin(options: UnoPostcssPluginOptions) {
 
     for (let i = 0; i < entries.length; i += BATCH_SIZE) {
       const batch = entries.slice(i, i + BATCH_SIZE)
-      promises.push(...batch.map(async ({ path: file, mtimeMs }) => {
+      promises.push(...batch.map(async (file) => {
         result.messages.push({
```

Comment:
is this required? I mean, shouldn't be added only if the file has changed?

Reply:
hmm, would moving `result.messages.push` after the `fileMap` check be a better option?

Reply:
no idea, I'm just asking...

Reply:
it makes sense to move it (at least to me), and tests seem to pass, so i'll push it